From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

carolinacmoravia committed:
C9575: Wrong location of HTML bookmark

Hello, @RenaShahMSFT, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.
   
Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

carolinacmoravia committed:
C9718:  Inconsistent table format: extra brealine required

Hello, @bryanla,

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  The incluse should be separated from the rest of the text. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 
Many thanks in advance.

tdykstra committed:
cleanup

paulathurman committed:
edit pass: five-more-media-services-articles

andredm7 committed:
Azure Stack 1712 release notes work

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into five-more-media-services-articles

kpiteira committed:
Removed old portal references for auth docs

kpiteira committed:
Fixed some acrolynx issues

kpiteira committed:
Fixed Note and title issues

paulathurman committed:
edit pass: five-more-media-services-articles

itechedit committed:
edit pass: devtest-lab-integrate-ci-cd-vsts

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into five-more-media-services-articles

andredm7 committed:
More changes

paulathurman committed:
edit pass: five-more-media-services-articles

carolinacmoravia committed:
Update msi-how-to-use-vm-msi-sign-in.md

Hello, @BryanLa,
I've checked with support team and this two extra spaces should fix this issue without modifying source content. Thank you.

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into five-more-media-services-articles

paulathurman committed:
edit pass: five-more-media-services-articles

paulathurman committed:
edit pass: five-more-media-services-articles

itechedit committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into devtest-lab-integrate-ci-cd-vsts

carolinacmoravia committed:
C9980: Incosistent format due to extra spaces

Hello, @rayne-wiselman,

Localization team has reported source content issue that causes localized version to have different format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

itechedit committed:
edit pass: devtest-lab-integrate-ci-cd-vsts

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

v-geberr committed:
Antonio Azevedo issue

v-geberr committed:
link

kpiteira committed:
Addressed PR comments

Commit:
added first screenshot

dominicbetts committed:
Update Dreamspark link

andredm7 committed:
Changes

tdykstra committed:
Add alternative to warning message

tdykstra committed:
add file connector link

andredm7 committed:
Changes

JasonWHowell committed:
Update concepts-limits.md

MicrosoftGuyJFlo committed:
Resubmitting overwritten redirect

Commit:
added updated screenshots

andredm7 committed:
Changes

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

kpiteira committed:
Fix some Acrolynx issues

kpiteira committed:
Removed a mention of classic portal

itechedit committed:
Update devtest-lab-integrate-ci-cd-vsts.md

MicrosoftGuyJFlo committed:
Resubmitting overwritten redirect

sanjeev3 committed:
Update quick-start.md

sanjeev3 committed:
Update quick-start.md

sanjeev3 committed:
Update credentials.md

sanjeev3 committed:
Update credentials.md

Commit:
followed matt's protocol for screenshots

neilpeterson committed:
new doc and toc

andredm7 committed:
Changes

kpiteira committed:
Fixing Image/Text alignment

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

markgalioto committed:
removing references to Classic portal in vault upgrade article

Ja-Dunn committed:
Merge pull request #30865 from MicrosoftGuyJFlo/MFARedirect

Resubmitting overwritten redirect

JasonWHowell committed:
Update concepts-limits.md

Ja-Dunn committed:
Merge pull request #30558 from kpiteira/azme-rm-oldportal-auth

Removed old portal references for auth docs

Ja-Dunn committed:
Merge pull request #30884 from sanjeev3/master

Updated Quick Start and Credentials Pages

YuriDio committed:
adding new article

Commit:
ToC and title updates.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into pbie0104

raiye committed:
Update cloud-services-guestos-msrc-releases.md

neilpeterson committed:
updates

Ja-Dunn committed:
Merge pull request #30888 from markgalioto/ab-1-3-2018

removing references to Classic portal in vault upgrade article

raiye committed:
Update cloud-services-guestos-update-matrix.md

Commit:
Remove question mark from title.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into pbie0104

neilpeterson committed:
toc update

Ja-Dunn committed:
Merge pull request #30855 from JasonWHowell/patch-4

PostgreSQL: Update concepts-limits.md

Lisaco88 committed:
Merge pull request #30890 from raiye/patch-3

Update cloud-services-guestos-msrc-releases.md

Lisaco88 committed:
Merge pull request #30892 from Minewiskan/pbie0104

Pbie0104

neilpeterson committed:
updates

Commit:
deleted old png files

sanjeev3 committed:
Update toc.yml

ShubhaVijayasarathy committed:
adding correct file format

sanjeev3 committed:
Create content-moderator-helper-quickstart-dotnet.md

Lisaco88 committed:
Merge pull request #30891 from raiye/patch-4

Update cloud-services-guestos-update-matrix.md

sanjeev3 committed:
Create image-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update content-moderator-helper-quickstart-dotnet.md

sanjeev3 committed:
Update toc.yml

ganesr committed:
Update create-vm-accelerated-networking-powershell.md

ShawnJackson committed:
Merge pull request #30649 from itechedit/devtest-lab-integrate-ci-cd-vsts

edit pass: devtest-lab-integrate-ci-cd-vsts

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

ganesr committed:
Update virtual-machines-common-sizes-general.md

ShawnJackson committed:
Merge pull request #30612 from paulathurman/five-more-media-services-articles

edit pass: Five more media services articles

ganesr committed:
Update create-vm-accelerated-networking-powershell.md

ganesr committed:
Update create-vm-accelerated-networking-powershell.md

sanjeev3 committed:
Create text-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update toc.yml

Lisaco88 committed:
Merge pull request #30798 from JimacoMS2/dps-fix-csharp-tpm-images

Fix screenshots in DPS csharp simulated TPM device quick start

andredm7 committed:
Changes

andredm7 committed:
Changes

andredm7 committed:
Changes

neilpeterson committed:
updates

Lisaco88 committed:
Merge pull request #30896 from ShubhaVijayasarathy/docchanges

adding correct file format

sanjeev3 committed:
Update content-moderator-helper-quickstart-dotnet.md

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update text-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update text-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update content-moderator-helper-quickstart-dotnet.md

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

neilpeterson committed:
toc update

andredm7 committed:
Changes

neilpeterson committed:
update

Juliako committed:
Update media-services-rest-create-contentkey.md

Sogup committed:
Added VM backup FAQ

dominicbetts committed:
Address PR feedback

JeffGoldner committed:
Important information for 1712 security patch

Signed-off-by: JeffGoldner <JeffGo@microsoft.com>

Commit:
updating URL to script action to the new one from engineering

sanjeev3 committed:
Create image-lists-quickstart-dotnet.md

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update image-lists-quickstart-dotnet.md

sanjeev3 committed:
Update image-lists-quickstart-dotnet.md

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into UpdatesJan2018_2

YuriDio committed:
adding new article - update

andredm7 committed:
Changes

YuriDio committed:
adding new article - update 2

andredm7 committed:
Changes

YuriDio committed:
fixing TOC

cynthn committed:
Update virtual-machines-common-accelerated-maintenance.md

Lisaco88 committed:
Merge pull request #30362 from carolinacmoravia/patch-516

C9575: Wrong location of HTML bookmark

Lisaco88 committed:
Merge pull request #30643 from carolinacmoravia/patch-529

C9980: Incosistent format due to extra spaces

ktoliver committed:
Merge pull request #30915 from Blackmist/freshness

updating URL to script action to the new one from engineering

ktoliver committed:
Merge pull request #30914 from JeffGoldner/master

Important information for 1712 security patch

cwatson-cat committed:
Updated redirect path for retired topic stream-analytics/stream-analytics-run-a-job

ktoliver committed:
Merge pull request #30913 from Sogup/patch-21

Added VM backup FAQ

ktoliver committed:
Merge pull request #30911 from Juliako/patch-1

Update media-services-rest-create-contentkey.md

cynthn committed:
Update virtual-machines-common-accelerated-maintenance.md

tdykstra committed:
change supported to available

ktoliver committed:
Merge pull request #30898 from ganesr/patch-1

Update create-vm-accelerated-networking-powershell.md

ktoliver committed:
Merge pull request #30816 from dominicbetts/fix-dreamspark-url

Update Dreamspark link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

ktoliver committed:
Merge pull request #30899 from ganesr/patch-2

Update virtual-machines-common-sizes-general.md

andredm7 committed:
Changes

Lisaco88 committed:
Changing H1 to sentence case.

ktoliver committed:
Merge pull request #30907 from sanjeev3/master

New Quickstarts and TOC entries

Lisaco88 committed:
Merge pull request #30918 from cynthn/patch-53

Update virtual-machines-common-accelerated-maintenance.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #30889 from YuriDio/UpdatesJan2018_2

adding new article

Lisaco88 committed:
Merge pull request #30738 from v-geberr/0103-import-version

Antonio Azevedo issue

Lisaco88 committed:
Merge pull request #30544 from andredm7/andredm1221171

Azure Stack 1712 release notes work

v-alje committed:
Merge pull request #30926 from MicrosoftDocs/FromPublicRepo

From public repo

rmca14 committed:
Merge pull request #30375 from carolinacmoravia/patch-519

C9718:  Inconsistent table format: extra breakline required

ktoliver committed:
Merge pull request #30920 from cwatson-cat/1-5-18-ClassicClnUp-SA-redirect

Updated redirect path for retired topic stream-analytics/stream-analy…

Lisaco88 committed:
Merge pull request #30597 from tdykstra/external

clean up external file and external table

rmca14 committed:
Merge pull request #30887 from neilpeterson/azure-files-kubernetes-pv

new doc and toc
